src/proto/payload.proto

```diff
@@ -1,8 +1,12 @@
-syntax = "proto3";
-package pb;
+syntax = "proto2";
 
-message NoiseHandshakePayload {
-	bytes identity_key = 1;
-	bytes identity_sig = 2;
-	bytes data = 3;
+message NoiseExtensions {
+    repeated bytes webtransport_certhashes = 1;
 }
+
+message NoiseHandshakePayload {
+  optional bytes identity_key = 1;
+  optional bytes identity_sig = 2;
+  optional bytes data         = 3;
```

Comment:
According to https://github.com/libp2p/specs/pull/453/files this should also be removed

Reply:
Yes, but I didn't want to make the breaking change here. This updates the api to make clients no longer able to send early data, but they can still receive early data.